docs/learn-parachains.md

```diff
@@ -5,88 +5,105 @@ sidebar_label: Parachains
-communicate with other parachains through [XCMP](learn-crosschain).
-
-Parachains are maintained by a network maintainer known as a [collator](learn-collator). The role of
-the collator node is to maintain a full-node of the parachain, retain all necessary information of
-the parachain, and produce new block candidates to pass to the Relay Chain validators for
-verification and inclusion in the shared state of Polkadot. The incentivization of a collator node
-is an implementation detail of the parachain. They are not required to be staked on the Relay Chain
-or own DOT tokens unless stipulated to do so by the parachain implementation.
+the validators of the Relay Chain. Most commonly a parachain will take the form of a blockchain, but
+there is no specific need for them to be actual blockchains. They take their name from the concept
+of parallelized chains that run parallel to the Relay Chain. Due to their parallel nature, they are
+able to parallelize transaction processing and achieve scalability of the
+{{ polkadot: Polkadot :polkadot }} {{ kusama: Kusama :kusama }} system. They
+[share in the security](https://wiki.polkadot.network/docs/en/learn-security) of the entire network
+and can communicate with other parachains through
+[XCMP](https://wiki.polkadot.network/docs/en/learn-crosschain).
+
+Parachains are maintained by a network maintainer known as a
+[collator](https://wiki.polkadot.network/docs/en/learn-collator). The role of the collator node is
+to maintain a full-node of the parachain, retain all necessary information of the parachain, and
+produce new block candidates to pass to the Relay Chain validators for verification and inclusion in
+the shared state of Polkadot. The incentivization of a collator node is an implementation detail of
+the parachain. They are not required to be staked on the Relay Chain or own DOT tokens unless
+stipulated to do so by the parachain implementation.
 
 The Polkadot Host (PH) requires that the state transitions performed on parachains to be specified
 as a Wasm executable. Proofs of new state transitions that occur on a parachain must be validated
 against the registered state transition function (STF) that is stored on the Relay Chain by the
-validators before Polkadot acknowledges a state transition has occurred on a parachain. The only
-constraint to the logic that a parachain is allowed to implement is that it must be verifiable by
-the Relay Chain validators. Verification most commonly takes the form of a bundled proof of a state
-transition known as a Proof-of-Verification (PoV) block, which is submitted to the validators from
-one or more of the parachain collators to be checked.
+validators before {{ polkadot: Polkadot :polkadot }} {{ kusama: Kusama :kusama }} acknowledges a
+state transition has occurred on a parachain. The only constraint to the logic that a parachain is
+allowed to implement is that it must be verifiable by the Relay Chain validators. Verification most
```

Comment:
```suggestion
state transition has occurred on a parachain. The key constraint regarding the logic of a parachain is
that it must be verifiable by the Relay Chain validators. Verification most
```
I believe there are other constraints here, e.g. in terms of relying on relay chain finalization,  block production time, etc.